bigbluebutton-html5/imports/ui/services/auth/index.js

```diff
@@ -155,6 +155,16 @@ class Auth {
         if (User.validated === true) {
           computation.stop();
           clearTimeout(validationTimeout);
+
+          if (User.ejected) {
```

Comment:
You can move this above of the `if(User.isvalidated === true)` since a ejected user always would be rejected.